### ISSUE TITLE
Error attempting to play tracks via Google's Unlimited Subscriptions

### ISSUE THREAD

**Issue by lyndsysimon** [kbhomes/radiant-player-mac#292]
I've added an album to My Library that I don't own, but I can access through my active Google Music subscription. It plays properly in Chrome, Safari, and Firefox, but when I attempt to play those songs through Radiant Player I get a "Blocked Plug-in".

![screen shot 2015-03-24 at 9 30 48 am](https://cloud.githubusercontent.com/assets/131414/6802762/4f11a31c-d209-11e4-9e32-80f7bd277ec9.png)

**Comment by lyndsysimon** [comment]
I removed my install (of v1.3.1), and cloned the repo (commit 7806af2d446e8f3799e1c9e89ec1300d3437956a). After building, it asked me to upgrade Flash in Safari, which I did. The built version works fine now - I have no way of knowing if that's because of the Flash update or because of my using the bleeding edge version.

If it's the former, then I'd still call this a bug since it didn't tell me what was wrong. If it was the latter, then this should be closed upon the next release.
